### PR TITLE
PHP 7.4 (don't merge, before switch over)

### PR DESCRIPTION
**Pull request by cableman**
#### Link to ticket

https://jira.itkdev.dk/browse/ER-1393

#### Description

Updates to the site to make it compatible with php 7.4.

#### Screenshot of the result

Screenshot 2022-10-26 at 12 56 05

#### Checklist

- [x] My code passes our static analysis suite.
- [x] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change 
should be exempt from the list.

#### Additional comments or questions

N/A

### CODE REVIEW

File: sites/all/modules/ereol_article/ereol_article.module [hunk 1]
```diff
@@ -57,7 +57,7 @@ function ereol_article_preprocess_node(&$variables) {
       '#offset' => 0,
       '#items' => array(),
       '#placeholder' => array(
-        '#theme' => 'reol_news_paragraph_placeholder',
+        '#theme' => 'ereol_article_placeholder',
```
Comment (rimi-itk) [inline sites/all/modules/ereol_article/ereol_article.module:60]: Has this never worked? Or worked incorrectly?

File: sites/all/modules/reol_base/reol_base.field.inc [hunk 2]
```diff
@@ -101,7 +101,7 @@ function reol_base_field_formatter_view($entity_type, $entity, $field, $instance
           $pub_prod = publizon_load($entity->localId);
           // If not on publizon, do nothing.
           if (!$pub_prod) {
-            continue;
+            break;
```
Comment (rimi-itk) [inline sites/all/modules/reol_base/reol_base.field.inc:104]: Semantic change. Do we really want to break out of the loop?

File: sites/all/modules/reol_base/reol_base.install [hunk 3]
```diff
@@ -871,3 +871,10 @@ function reol_base_update_7172() {
 function reol_base_update_7173() {
   module_enable(array('elements'));
 }
+
+/**
+ * Enable tiny png module.
```
Comment (rimi-itk) [inline sites/all/modules/reol_base/reol_base.install:876]: Should be “Remove …”.

**Review by rimi-itk — COMMENTED** [review]
LGTM. A few comments/questions should be addressed.

File: sites/all/modules/ereol_article/ereol_article.module (hunk 1 above)
Comment (cableman) [reply sites/all/modules/ereol_article/ereol_article.module:60]: Never worked I think....

File: sites/all/modules/reol_base/reol_base.field.inc (hunk 2 above)
Comment (cableman) [reply sites/all/modules/reol_base/reol_base.field.inc:104]: PHP 7.4 complained about mis-used `continue` statement.... it's not the loop that break's but the switch statement (which is the only construct inside that loop) that break. So here break will have the samme effect as the continue before that php 7.4 did throw notices about.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**